index.html

```diff
@@ -323,7 +324,7 @@ <h1 class="fisto">Contact me</h1><br>
   <div class="popup-container">
     <div class="popup">
       <div class="popup-header">
-        <h2 id="popup-title"> </h2>
+        <h2 id="popup-title"></h2>
         <i id="close-popup" class="fas fa-times"></i>
       </div>
       <div class="popup-details">
```

Comment:
Update your ``README`` file with the required information by the template:
<img width="1440" alt="Screen Shot 2021-11-11 at 5 01 25" src="https://user-images.githubusercontent.com/22258274/141288054-9e043dcc-d3a6-459e-8f25-25c1996392d8.png">